pkg/types/encoding/json/encoding.go

```diff
@@ -56,9 +56,6 @@ func matchesToJSONMatches(matches types.Matches) []jsonMatch {
 			jm.Leaf = 1
 		} else {
 			jm.AllowChildren = 1
-		}
-
-		if !m.IsLeaf || strings.ContainsRune(jm.ID, '*') {
```

Comment:
Looks logical that we should enable Expandable for branches with globs - but that's not how graphite-web works. It sets expandable=true for all branches - see for reference https://github.com/graphite-project/graphite-web/blob/02bc0c836269e07b38175fbfe3b5689afc87b4fa/webapp/graphite/metrics/views.py#L315-L324 and https://github.com/graphite-project/graphite-web/blob/02bc0c836269e07b38175fbfe3b5689afc87b4fa/webapp/graphite/browser/views.py#L86-L95

Reply:
Is this a separate change from adding `/metric/expenad`?

Reply:
Yes, it is. We have separate ticket for that. I can remove it if you hesitate.

Reply:
Probably, you're right, will exclude it and create separate PR

Reply:
Removed